src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,253 @@
+          <div className='description'>
+            <h2>
+              Stream Plethora
+              <div></div>of Movies, Series & Reviews
+              <div></div>around Autoculture
+            </h2>
+          </div>
+        </div>
+      </div>
+      <div className='categories'>
+        <div className='category-movies'>
+          <div className='category-name'>Movies</div>
+          <span
+            className='material-symbols-outlined arrow movie-left-abs'
+            onClick={() => movieHandleScroll('left')}
+            style={{ display: !isScrolled && 'none' }}
+          >
+            arrow_back_ios
+          </span>
+          <div className='category-cards' ref={movieContainerRef}>
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+            <VCard thumbnail={picsum} title={'Ford vs Ferrari'} />
+          </div>
+          <span
+            className='material-symbols-outlined arrow movie-right-abs'
+            onClick={() => movieHandleScroll('right')}
+          >
+            arrow_forward_ios
+          </span>
+        </div>
+
+        <div className='space'></div>
+
+        <div className='category-series'>
+          <div className='category-name'>Series</div>
+          <span
+            className='material-symbols-outlined arrow series-left-abs'
+            onClick={() => seriesHandleScroll('left')}
+            style={{ display: !isScrolled && 'none' }}
+          >
+            arrow_back_ios
+          </span>
+          <div className='category-cards' ref={seriesContainerRef}>
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+            <VCard thumbnail={spicsum} title={'Cars'} />
+          </div>
```

Comment:
we can do .map()

Reply:
yupp will do that, added more components just for demo purpose, once I update db, will pull everything from there
Thanks for the suggestion